### PR TITLE
OC-11318 Support custom item value display for Visit-Based events in Participant Matrix

### PR DESCRIPTION
**Pull request by jkeremian**
Support custom item value display for Visit-Based events in Participant Matrix

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java [hunk 1]
```diff
@@ -76,6 +77,11 @@
     // request attribute for a discrepancy note
     public final static String ENROLLMENT_NOTE = "enrollmentNote";
     public final static String COMMON = "common";
+    public final static String OPEN_BRACKET = "[";
+    public final static String CLOSE_BRACKET = "]";
+    public final static String DOT_ESCAPED = "\\.";
+
+    public final static String visitBasedEventItempath=CoreResources.getField("visitBasedEventItem");
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java]: can you commit this property in the datainfo.properties file with some example value for documentation purposes?

Comment (jkeremian) [reply web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java]: I already added  the property sample in the story in the comment https://jira.openclinica.com/browse/OC-11318. Also added some instructions on how to test some criteria.
I don't want to check in my datainfo.properties file to master.

Comment (svadla-oc) [reply web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java]: sounds good, thanks

**Comment by svadla-oc** [comment]
👍

**Pull request merged**